flutter/lib/src/default_integrations.dart

```diff
@@ -364,23 +364,21 @@ class LoadReleaseIntegration extends Integration<SentryFlutterOptions> {
   @override
   FutureOr<void> call(Hub hub, SentryFlutterOptions options) async {
     try {
-      if (!kIsWeb) {
-        if (options.release == null || options.dist == null) {
-          final packageInfo = await _packageLoader();
-          var name = packageInfo.packageName;
-          if (name.isEmpty) {
-            // Not all platforms have a packageName.
-            // If no packageName is available, use the appName instead.
-            name = _cleanAppName(packageInfo.appName);
-          }
+      if (options.release == null || options.dist == null) {
```

Comment:
merge main into this branch

Reply:
This is a change of this branch. The `isWeb` check got removed in this branch and thus the indentation.